kafka/activity/README.md

```diff
@@ -0,0 +1,92 @@
+<!--
+title: Kafka
+weight: 4616
+-->
+
+# Kafka Activity
+
+This activity publishes messages on a topic in a Kafka cluster.
+
+### Flogo CLI
+```bash
+flogo install github.com/project-flogo/messaging-contrib/kafka/activity
+```
+
+## Configuration
+
+### Settings: 
+| Name       | Type   | Description
+|:---        | :---   | :---   
+| connection | any    | The connection object which is use to connect to pulsar - ***REQUIRED***
+| topic      | string | The Kafka topic on which to place the message - ***REQUIRED***
+| user       | string | If connecting to a SASL enabled port, the user id to use for authentication
```

Comment:
should user, pass and truststore be part of connection? Maybe have a connection section with settings?

Reply:
Shouldn't that be the case? If you have to use the truststore  for connecting ; that should be used for every connection.

Reply:
Looking at your settings object, it does not have those
```
type Settings struct {
	Connection connection.Manager `md:"connection,required"`
	Topic      string             `md:"topic,required"` // The Kafka topic on which to place the message
}
```

So it looks like just the documentation is incorrect.  Maybe having "Connection" section that has the settings or reference the connection readme.